### DISCUSSION TITLE
Can I add data preprocessing to a rtmdet model when i convert it to onnx

### DISCUSSION THREAD

**Discussion by foshyjoshy** [open-mmlab/mmdeploy#2804] [Q&A]
I am converting my trained model using tools/deploy.py to onnx so im able to run it using onnxruntime. Can i include the pipeline pre processing to my model. Im looking to resize and pad my input image as part of the network to match the models fixed input.

**Comment by ayush-AkulaTech** [comment]
@foshyjoshy Hi how are you going with this? I'm in a similar situation with a different model, I am wondering if the preprocessing pipeline has to now be done manually before inferencing images or is that handled by the inference SDK? For example If in my model config I am center cropping my images to 224x224, then is that something I have to now do manually to the images before inference?
